### PR TITLE
Fix: Fixed check for admin

### LINKED ISSUE #13394: Bug: App crashes when dragging a file while app is running with a high integrity level
### Description

Crashed after opening app and draging a file. No other actions. App is not running as admin, and is not set as the default file manager. Sometimes it happened to work well with no crash for drag & drop but I am not sure about the reason.

- [ ] 

### Steps To Reproduce

1. open Files with a high integrity level
2. drag a file or tab
3. the app crashes

### Requirements

- Detect and disable drag & drop on all trust levels that don't support it

### Files Version

2.5.29.0

### Windows Version

10.0.19045.3448

### Log File

[debug.log](https://github.com/files-community/Files/files/12653254/debug.log)

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   For #13394...

**Validation**
How did you test these changes?
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Helpers/Environment/ElevationHelpers.cs [hunk 1]
```diff
@@ -21,8 +21,8 @@ public static bool IsElevationRequired(string filePath)
 
 		public static bool IsAppRunAsAdmin()
 		{
-			using WindowsIdentity identity = WindowsIdentity.GetCurrent();
-			return new WindowsPrincipal(identity).IsInRole(WindowsBuiltInRole.Administrator);
+			WindowsIdentity identity = WindowsIdentity.GetCurrent();
```
Comment (hishitetsu) [inline src/Files.App/Helpers/Environment/ElevationHelpers.cs:24]: Is there no need for `using`?

Comment (yaira2) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:24]: It works without it but I'm not sure if it makes a difference.

Comment (yaira2) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:24]: Anyways, I think we need some more testing to see if this change unblocks users that have UAC turned off.

Comment (hishitetsu) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:24]: I believe it is safer to leave the `using`. But I can't determine whether the modification on line 25 is correct or not.

**Pull request merged**